scenarioo-client/test/protractorE2E/specs/list_scenarios.ts

```diff
@@ -1,79 +1,78 @@
 'use strict';
-import {scenario, step, useCase} from "scenarioo-js";
 
-var scenarioo = require('scenarioo-js');
-var pages = require('./../webPages');
-var NUMBER_OF_USE_CASES = 4;
-var NUMBER_OF_SCENARIOS = 4;
-var COMPARISON_PROJECTSTART = 'To Projectstart';
-var SECOND_USE_CASE = 1;
-var SCENARIO_WITH_HIGHEST_DIFF = 'Find page title unique directly';
+import { scenario, step, useCase } from 'scenarioo-js';
+import * as Utils from '../util/util';
+import HomePage from '../webPages/homePage';
+import UsecasePage from '../webPages/usecasePage';
+import ScenarioPage from '../webPages/scenarioPage';
+import NavigationPage from '../webPages/navigationPage';
+
+const NUMBER_OF_USE_CASES = 4;
+const NUMBER_OF_SCENARIOS = 4;
+const COMPARISON_PROJECTSTART = 'To Projectstart';
+const SECOND_USE_CASE = 1;
+const SCENARIO_WITH_HIGHEST_DIFF = 'Find page title unique directly';
 
 useCase('List scenarios')
     .description('After clicking on a use case, the user is presented with a list of all scenarios in this use case.')
-    .describe(function () {
-
-        var homePage = new pages.homePage();
-        var useCasePage = new pages.usecasePage();
-        var scenarioPage = new pages.scenarioPage();
-        var navigationPage = new pages.navigationPage();
+    .describe(() => {
 
-        beforeEach(function () {
-            new pages.homePage().initLocalStorage();
+        beforeEach(async () => {
+            await Utils.startScenariooRevisited();
         });
 
         scenario('Expand all, collapse all on scenario page')
-            .it(function () {
-                homePage.goToPage();
-                step('select a use case from the use case list');
-                homePage.assertPageIsDisplayed();
-                homePage.assertUseCasesShown(NUMBER_OF_USE_CASES);
-                homePage.selectUseCase(SECOND_USE_CASE);
-                step('select a scenario in the scenario list');
-                useCasePage.selectScenario(0);
-                step('all pages are collapsed by default, "expand all" button is visible');
-                scenarioPage.expectOnlyExpandAllButtonIsDisplayed();
-                scenarioPage.toggleShowAllStepsOfPage(0);
-                step('"expand all" button and "collapse all" button are both visible');
-                scenarioPage.expectExpandAllAndCollapseAllButtonBothDisplayed();
-                scenarioPage.toggleShowAllStepsOfPage(1);
-                step('Only "collapse all" visible');
-                scenarioPage.expectOnlyCollapseAllButtonIsDisplayed();
+            .it(async () => {
+                await Utils.navigateToRoute('/');
```

Comment:
I think `HomePage.goToPage()` was nicer than a direct navigateToRoute

Reply:
this has been changed in many other places.

Reply:
Agree, done.